initial-promises/rootServerInitialPromises/cfengine-nova/distributePolicy/logrotate.conf/rudder.rhel

```diff
@@ -38,7 +38,7 @@
         create 640 root root
         delaycompress
         postrotate
-          /etc/init.d/rsyslog reload > /dev/null
+          /bin/kill -HUP `cat /var/run/syslogd.pid 2> /dev/null` 2> /dev/null || true
```

Comment:
i'm not sure of that, you'd need to check that rsyslog is not using the syslogd.pid file